lib/constable/services/hub_profile_provider.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Constable.Services.HubProfileProvider do
+  alias Constable.Services.ProfileProvider
+  @behaviour ProfileProvider
+  use Memoize
+
+  @default_image_url "/images/ralph.png"
```

Comment:
I don’t see a `ralph.png` in the [`images` directory here in Constable](https://github.com/thoughtbot/constable/tree/master/assets/static/images)…is this coming from Hub?

Reply:
It was added to the `images` directory [as part of this PR](https://github.com/thoughtbot/constable/pull/743/files#diff-4c5e766b6bbf2c9c11e874e97076009f)

Reply:
Whoops! Thanks for clarifying. I sometimes miss those in GitHub diffs because they are collapsed.